tests/tools/kopiarunner/kopia_snapshotter.go

```diff
@@ -479,3 +481,23 @@ func certKeyExist(ctx context.Context, tlsCertFile, tlsKeyFile string) error {
 func errIsACLEnabled(stdErr string) bool {
 	return strings.Contains(stdErr, aclEnabledMatchStr)
 }
+
+// GetRepositoryStatus returns the repository status in JSON format.
+func (ks *KopiaSnapshotter) GetRepositoryStatus() cli.RepositoryStatus {
+	a1, _, _ := ks.Runner.Run("repository", "status", "--json")
+
+	var rs cli.RepositoryStatus
+	if err := json.Unmarshal([]byte(a1), &rs); err != nil {
+		panic(err)
```

Comment:
Change the function return values to return `(cli.RepositoryStatus, error)` and then return the error instead of `panic`ing.

Reply:
Done